homeassistant/components/greeneye_monitor/sensor.py

```diff
@@ -237,7 +240,7 @@ def native_value(self) -> float | None:
             * self._counted_quantity_per_pulse
             * self._seconds_per_time_unit
         )
-        return result
+        return cast(float, result)
```

Comment:
The underlying library is not typed, so mypy needs `cast`.

Reply:
For some reason this one fails on my machine but not in CI
```suggestion
        return result
```